### PR TITLE
Abhiramreddy/registration update

### PR DESCRIPTION
**Pull request by nrabhiram**
Thanks a lot for the challenge! Hope my finalist-entry will be accepted.

### CODE REVIEW

File: content/finalist/ttjc@finalist@nrabhiram/index.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+Links:
+* Project: (https://forkify1.netlify.app/)
+* Github repo: (https://github.com/nrabhiram/forkify)
+
+### blogs
+
+1. [Imports and Exports](https://dev.to/nrabhiram/imports-exports-52fg)
+2. [Why is React So Popular](https://dev.to/nrabhiram/why-is-react-so-popular-1oa1)
+3. [React-Redux](https://dev.to/nrabhiram/react-redux-1622)
```
Comment (gkanishk) [inline content/finalist/ttjc@finalist@nrabhiram/index.md:103]: provide description

File: content/finalist/ttjc@finalist@nrabhiram/index.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
+---
+title: Abhiram Reddy
+type: "finalist"
+name: "Abhiram Reddy"
+portfolio: "https://abhiramreddy.netlify.app/"
+github-link: "https://github.com/nrabhiram"
+linkedin-link: "https://www.linkedin.com/in/abhiram-reddy-23285b196/"
+looking-for: "job"
+image-link: "https://media-exp1.licdn.com/dms/image/C5603AQFYdU_qNBpPYQ/profile-displayphoto-shrink_200_200/0?e=1598486400&v=beta&t=Wl5ltoM1oIVNo4rIPWwbW_qHeZ1gvW0crsSMrPKN228"
```
Comment (gkanishk) [inline content/finalist/ttjc@finalist@nrabhiram/index.md:9]: Please name the variables as follows:
```
github_link:
linkedin_link:
looking_for:
image_link:

```

**Comment by nrabhiram** [comment]
Got it @gkanishk. I have made the required changes.

**Comment by gkanishk** [comment]
Awesome kindly provide details here #1033

**Pull request merged**